pyseir/icu/infer_icu.py

```diff
@@ -0,0 +1,133 @@
+import json
+import structlog
+import os.path
+from dataclasses import dataclass
+from enum import Enum
+
+import pandas as pd
+
+from pyseir import DATA_DIR
+import libs.datasets.combined_datasets as combined_datasets
+from covidactnow.datapublic.common_fields import CommonFields
+
+logger = structlog.get_logger()
+
+
+class ICUConfig:
+    LOOKBACK_DATE = pd.Timestamp.today() - pd.Timedelta(days=91)
+
+
+class WeightsBy(Enum):
+    # Distribute unattributed covid icu patients based on county population
+    POPULATION = os.path.join(DATA_DIR, "population_weights_via_fips.json")
+    # Distribute unattributed covid icu patients based on a county's fraction of the last month's
+    # total cases for that state (pre-trained out of band).
+    ONE_MONTH_TRAILING_CASES = os.path.join(DATA_DIR, "one_month_trailing_weights_via_fips.json")
+
+
+@dataclass
+class LinearRegressionCoefficients:
+    m_hospitalized: float = 0.2885
```

Comment:
So, 28% of hospitalizations result in ICU cases based on your current regression?